### ISSUE TITLE
[Bug] NFT display error

### ISSUE THREAD

**Issue by chiadev** [Chia-Network/chia-blockchain#13780]
### What happened?

When opening the NFT browse, NFT cannot be displayed, and the error message is prompted

![image](https://user-images.githubusercontent.com/22480/199189810-76786315-bfe2-4e06-9299-ef3b66f746ad.png)

![image](https://user-images.githubusercontent.com/22480/199189837-9bdcb059-fcb7-4c44-9be7-2627f2c0a0ad.png)
### Version

1.6.1rc5

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
......
2022-11-01T15:45:22.325 full_node chia.full_node.mempool  : WARNING  Fee estimator error
2022-11-01T15:45:22.325 full_node chia.full_node.mempool  : WARNING  Fee estimator error
2022-11-01T15:45:22.325 full_node chia.full_node.mempool  : WARNING  Fee estimator error
2022-11-01T15:45:22.325 full_node chia.full_node.mempool  : WARNING  Fee estimator error
2022-11-01T15:45:22.325 full_node chia.full_node.mempool  : WARNING  Fee estimator error
2022-11-01T15:45:22.325 full_node chia.full_node.mempool_manager: INFO     Size of mempool: 664 spends, cost: 547658904684 minimum fee rate (in FPC) to get in for 5M cost tx: 0
2022-11-01T15:45:22.404 wallet chia.wallet.wallet_blockchain: INFO     Peak set to: 2762366 timestamp: 1667288689
2022-11-01T15:45:23.669 full_node chia.full_node.full_node: INFO       Finished signage point 35/64: CC: 268bd2257c351e756395ddd7033230825bd207af28e2ea90c59f5b98d560da84 RC: c5b979d3441127acbcf606ce82bffa2b5d0eeeab04ca2fc350f379b367fbba77 
2022-11-01T15:45:23.685 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
2022-11-01T15:45:32.183 full_node chia.full_node.full_node: INFO       Finished signage point 36/64: CC: eb608bd9d63a68e0110b8b942562df3e4cbb08b17fd8880fc5600f017c7e61bb RC: 0f6d4929e7a62e73f71ca06cea77514fafd095ac802091733828df3403860232 
2022-11-01T15:45:32.183 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
2022-11-01T15:45:41.291 full_node chia.full_node.full_node: INFO      Updated peak to height 2762367, weight 5741596648, hh bc9cd844e75b185ca61687bd7f3615aeaa4eef8be01a2a1dcd15c7e5c2a61758, forked at 2762366, rh: f1f03628e7385064ea5dca2470f844f8efe353eb4bfb3a2511d68f4a8cbaa4be, total iters: 11203262379641, overflow: False, deficit: 3, difficulty: 1816, sub slot iters: 137363456, Generator size: No tx, Generator ref list size: No tx
2022-11-01T15:45:41.306 wallet chia.wallet.wallet_blockchain: INFO     Peak set to: 2762367 timestamp: 1667288689
2022-11-01T15:45:42.118 full_node chia.full_node.full_node: INFO       Finished signage point 37/64: CC: dfdd22271e8c08ebfeffc98dc60487d69b636afae2060e92e75d54afca5523cf RC: 4ca70ab445df87a0fd33225988edb037463ecac813b7dabe7fa7c32881077dea 
2022-11-01T15:45:42.118 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
2022-11-01T15:45:50.757 full_node chia.full_node.full_node: INFO       Finished signage point 38/64: CC: 5ce9087769fefe5a15e905fb6a93647b685af52b64482673ca9f44fdda604936 RC: 8296e059e9795d7227bdf925faf5fb30ddb4bef484dcec1be0e10f9ffeadb4ac 
2022-11-01T15:45:50.757 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
2022-11-01T15:45:51.116 full_node chia.full_node.full_node: WARNING  Time for header validate: 0.015534162521362305
2022-11-01T15:45:51.163 full_node chia.full_node.full_node: INFO     Added unfinished_block 37980a808198f1f27c373e15a442675869c95c8d75ff6496588e74f215409678, not farmed by us, SP: 37 farmer response time: 8.9824, Pool pk xch1704ps5k6z9esdv64jntwhve4syraarylhsedwqhczjvuyjkxx8uqcyxzx4, validation time: 0.0156 seconds, pre_validation time 0.0156, cost: 5395389781, percent full: 49.049%
2022-11-01T15:45:59.436 full_node chia.full_node.full_node: INFO       Finished signage point 39/64: CC: c225eb0ec5b137841e46d618081772ef116a1f73b3a3b6301549849b2635c023 RC: b079c25cf72e51c0f87a4388834b8b63b3b8cbb229ca21e7e6359bcd19184443 
2022-11-01T15:45:59.452 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
2022-11-01T15:45:59.811 full_node full_node_server        : INFO     Cannot connect to host 108.52.109.104:8444 ssl:<ssl.SSLContext object at 0x0000021FA635BC40> [信号灯超时时间已到]
2022-11-01T15:46:08.280 full_node chia.full_node.full_node: INFO       Finished signage point 40/64: CC: 8decdb51f8f58e77e861578a55cb005a9ca93f4f7f708acad92cdd563c824f75 RC: 0e0465528a0065f77cf8dc3d4b3888cb0602d12ba24300179678ade29f10f3a3 
2022-11-01T15:46:08.280 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 0bc9fc0a59... Found 0 proofs. Time: 0.00000 s. Total 1 plots
```

**Comment by advlive** [comment]
Yes has error on NFT section request reload but last version now is ChiaSetup-latest-dev 1.6.1.rc4-dev63!!

**Comment by chiadev** [comment]
> 

If the bug has been solved, then I will wait for the official version to update again

**Comment by paninaro** [comment]
@chiadev can you include some NFT identifiers that were causing the issue? It's much easier to diagnose and fix the issue with concrete examples.

**Comment by advlive** [comment]
Already I install ChiaSetup-latest-dev 1.6.1.rc5-dev86 as in this NFT will show but not all of availability around 50% of NFT in wallet did not show and I think something still wrong .even node is sync ,wallet is sync just close app and re open again so long take times to sync show wallet 
after ChiaSetup-latest-dev 1.6.1.rc3-dev55 wallet issue is start

**Comment by Svetyr** [comment]
Hash is validated.1.6.1 Windows 10

**Comment by Svetyr** [comment]
![Снимок экрана 2022-11-05 105146](https://user-images.githubusercontent.com/100843682/200109744-8c6b47bc-f6e9-4bee-a045-f628bd0274dc.png)
1.6.1 Windows 10

**Comment by wallentx** [comment]
Closing since this should have been resolved in the 1.6.1 release, but do reopen if you see this again.
